assets/agenda/components/AgendaListItem.tsx

```diff
@@ -185,17 +194,27 @@ class AgendaListItem extends React.Component<any, any> {
 
                         {(isMobile || isExtended) && description && (
                             <div className="wire-articles__item__text">
-                                {item.es_highlight && item.es_highlight
+                                {item.es_highlight && item.es_highlight.definition_short
```

Comment:
would be better to just remove that extra condition, there might not be highlight for `definition_short` but then it would use the other one

Reply:
...and for other one dont need to show full description list... bcz of that i need this extra condition